src/MICore/Checksum.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MICore
+{
+    public class Checksum
+    {
+#if CORECLR
```

Comment:
Went with an enum.

---
In reply to: [64843632](https://github.com/Microsoft/MIEngine/pull/346#discussion_r64843632) [](ancestors = 64843632,64833308)